python3Packages.mautrix: 0.19.7 -> 0.19.9

NickCao opened the pull request:
Diff: https://github.com/mautrix/python/compare/refs/tags/v0.19.7...v0.19.9

Changelog: https://github.com/mautrix/python/releases/tag/v0.19.9

###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

NickCao committed:
python3Packages.mautrix: 0.19.7 -> 0.19.8

Diff: https://github.com/mautrix/python/compare/refs/tags/v0.19.7...v0.19.8

Changelog: https://github.com/mautrix/python/releases/tag/v0.19.8

sumnerevans commented:
We should go all the way to [0.19.9](https://github.com/mautrix/python/releases/tag/v0.19.9)

NickCao committed:
python3Packages.mautrix: 0.19.8 -> 0.19.9

Diff: https://github.com/mautrix/python/compare/refs/tags/v0.19.8...v0.19.9

Changelog: https://github.com/mautrix/python/releases/tag/v0.19.9

SuperSandro2000 commented:
@ofborg build v

NickCao commented:
Result of `nixpkgs-review pr 225267` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>mautrix-facebook</li>
    <li>mautrix-facebook.dist</li>
    <li>mautrix-googlechat</li>
    <li>mautrix-googlechat.dist</li>
    <li>mautrix-signal</li>
    <li>mautrix-signal.dist</li>
    <li>mautrix-telegram</li>
    <li>mautrix-telegram.dist</li>
    <li>python310Packages.mautrix (python310Packages.mautrix-appservice)</li>
    <li>python310Packages.mautrix.dist (python310Packages.mautrix-appservice.dist)</li>
    <li>python311Packages.mautrix (python311Packages.mautrix-appservice)</li>
    <li>python311Packages.mautrix.dist (python311Packages.mautrix-appservice.dist)</li>
  </ul>
</details>
